src/test/resources/edu/cornell/kfs/sys/businessobject/lookup/fixture/sub1/testlookupfile1.txt

```diff
@@ -0,0 +1 @@
+just a placeholder file for testing
```

Comment:
Probably not worth changing since it is already in prod, but I would suggest having the unit create these test files, and then delete them after the test have completed.  It feels better than keeping files like this in our code repo.  What are your thoughts on this?